gcloud-java-examples/src/main/java/com/google/gcloud/examples/resourcemanager/ResourceManagerExample.java

```diff
@@ -33,11 +33,9 @@
  * <p>This example creates, deletes, gets, and lists projects.
  *
  * <p> Steps needed for running the example:<ol>
- * <li>login using gcloud SDK - {@code gcloud auth login}.</li>
- * <li>compile using maven - {@code mvn compile}</li>
- * <li>run using maven - {@code mvn exec:java
- * -Dexec.mainClass="com.google.gcloud.examples.resourcemanager.ResourceManagerExample"
- * -Dexec.args="[list | [create | delete | get] projectId]"}</li>
+ * <li>log in using gcloud SDK - {@code gcloud auth login}.</li>
```

Comment:
Not related to this PR but could you remove the space after `<p>` on the previous line?